modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -0,0 +1,41 @@
+process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
+    tag '$gtf'
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::atlas-gene-annotation-manipulation=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/atlas-gene-annotation-manipulation%3A1.1.0--hdfd78af_0':
+        'quay.io/biocontainers/atlas-gene-annotation-manipulation:1.1.0--hdfd78af_0' }"
+
+    input:
+    path gtf
+    path fasta
+
+    output:
+    path "feature_annotation.tsv"   , emit: feature_annotation
+    path "versions.yml"             , emit: versions
+    path "*.fa.gz"                  , emit: filtered_cdna, optional: true
```

Comment:
If you decide on adding meta to your input gtf channel, I'd also add meta to the ouput channel definitions here.